share/spice/playing_cards/content.handlebars

```diff
@@ -0,0 +1,5 @@
+<div>
```

Comment:
Oops, I forgot about that (though I did remove the extra handlebars file). But that wasn't my problem. Here's my code right now; I'm still getting clickable tiles, even after setting ```detail``` to false.

```javascript
(function (env) {
    "use strict";
    env.ddg_spice_playing_cards = function(api_result){
        if(api_result.error || !api_result.cards) {
            return Spice.failed('playing_cards');
        }
        
        Spice.add({
            id: "playing_cards",
            name: "Deck of Cards",
            data: api_result.cards,
            meta: {
                sourceName: "DeckOfCards",
                sourceUrl: 'http://deckofcardsapi.com'
            },
            normalize: function(item) {
                return {
                    image: item.image
                };
            },
            templates: {
                group: 'movies',
                detail: false
            }
        });
     }
  }
(this));
```